### DISCUSSION TITLE
Can we integrate integrations supported by elastic?

### DISCUSSION THREAD

**Discussion by Blason** [Security-Onion-Solutions/securityonion#5613] [Unsupported Versions] [answered]
Hi Team,

Since SO is based on elastic wondering if all those integration mentioned as those are filebeat modules are supported by SO as well?
https://www.elastic.co/integrations?solution=all-solutions
Or if not by default can we add up those modules using filebeat and have them integrated with SO?

**Comment by dougburks** [comment, marked as answer]
We can't guarantee that all of those integrations work, but we did add some support for Filebeat modules recently:
https://docs.securityonion.net/en/2.3/filebeat.html#modules
